types/11ty__eleventy-img/package.json

```
@@ -0,0 +1,6 @@
+{
+    "private": true,
+    "dependencies": {
+        "@types/sharp": "^0.31.0"
```

These should get updated to reference the new bundled types
```suggestion
        "sharp": "^0.32.0"
```

---

I would say that belongs in a separate PR. It is only in `@11ty/eleventy-img` v3.1.0 which was released 4 days ago that it uses 0.32 - prior to that it was using the 0.31 range. The `@types/11ty__eleventy-img` module should mirror that and release a 3.1.0 which includes the 0.32 typings.